src/i-client-config.ts

```diff
@@ -2,6 +2,8 @@ import { AttributeType, Flag, IAssignmentLogger, IAsyncStore } from '@eppo/js-cl
 
 import { ServingStoreUpdateStrategy } from './isolatable-hybrid.store';
 
+export const DEFAULT_LOCAL_STORE_HEAD_START_MS = 10;
```

Comment:
I wonder if we want a more descriptive variable name, although I can't think of a good one.

`DEFAULT_PERSISENT_STORE_INIT_WINDOW`?

Either way I think we use the "persistent" messaging more than "local"